homeassistant/components/update/__init__.py

```diff
@@ -153,6 +159,15 @@ async def async_skip(entity: UpdateEntity, service_call: ServiceCall) -> None:
     await entity.async_skip()
 
 
+async def async_clear_skipped(entity: UpdateEntity, service_call: ServiceCall) -> None:
+    """Service call wrapper to validate the call."""
+    if entity.auto_update:
+        raise HomeAssistantError(
+            f"Clearing skipped update is not supported for {entity.name}"
+        )
```

Comment:
Should we fail here if the entity is not skipped?

Reply:
🤷 It will not trigger a state change. It's like turning on a light that is already on.